rest/route/task_generate.go

```diff
@@ -47,6 +52,29 @@ func (h *generateHandler) Run(ctx context.Context) gimlet.Responder {
 	if err := data.GenerateTasks(h.taskID, h.files); err != nil {
 		return gimlet.MakeJSONInternalErrorResponder(errors.Wrapf(err, "generating tasks for task '%s'", h.taskID))
 	}
+	t, err := task.FindOneId(h.taskID)
+	if err != nil {
+		return gimlet.MakeJSONInternalErrorResponder(errors.Wrapf(err, "getting task '%s'", h.taskID))
+	}
+	if t == nil {
+		return gimlet.MakeJSONErrorResponder(errors.Errorf("task '%s' not found", h.taskID))
+	}
+	ts := utility.RoundPartOfMinute(1).Format(units.TSFormat)
```

Comment:
There's some duplication between here and the cron populator, so I would turn this into a helper function in `units` that both can just call to make a job for one task.